packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -125,6 +139,15 @@ async function sendFeedback(): Promise<void> {
         <ErrorMessage class="text-xs" error="Please share contact info or details on how we can improve" />
       {:else if smileyRating === 2 && !hasFeedback}
         <WarningMessage class="text-xs" error="We would really appreciate knowing how we can improve" />
+      {:else}
```

Comment:
this will also appear with 'Love' if rating is 1 or 2 and feedback is done. I suppose it is not on purpose
```suggestion
      {:else if smileyRating > 2}
```